client/src/Movies/MovieCard.js

```diff
@@ -1,7 +1,27 @@
 import React from 'react';
 
 const MovieCard = props => {
-  return;
+  const { title, director, metascore, stars } = props.movie;
+  return (
+    
+    <div className="movie-card">
+      <h2>{title}</h2>
+      <div className="movie-director">
+        Director: <em>{director}</em>
+      </div>
+      <div className="movie-metascore">
+        Metascore: <strong>{metascore}</strong>
+      </div>
+      <h3>Actors</h3>
+
+      {stars.map(star => (
+        <div key={star} className="movie-star">
+          {star}
+        </div>
+      ))}
+      {/* <div className="save-button" onClick={props.onClick}>Save</div> */}
```

Comment:
you're already passing the save function down properly, all you have to do is uncomment this and reference the right one being passed down via props. note that you're passing the same function to one component from different sources since both MovieList and Movie render the MovieCard component, so both don't need to pass the save function down. you could only pass it down from App >> Movie >> MovieCard as you have it, and then reference it here as `props.saveMovie()` as it was passed down from Movie.js